app/main.py

```diff
@@ -116,6 +116,70 @@ def upload():
     else:
         flash('Only CSV file is accepted!', 'error')
         return redirect(url_for('index'))
+=======
```

Comment:
@pushkalkatara I think you forget to remove this

Reply:
You have to remove the entire part that is repeated twice.

Reply:
All you did was removed the `<<<<<<development` part. Please read my previous comment carefully